rmarkdowns/Week_2.Rmd

```diff
@@ -201,6 +245,18 @@ washington <- states %>%
 Now let's add that as a layer (the last `geom_sf` in our code) on our map, use the same fill color as the other states, and color the border red. We want this layer to be on top of the other layers on our map, so it should come after (below) the other `geom_sf` lines of our code. The order of our layers is very important!
 
 **Challenge 7: Update the code below with all of your changes.** You should be changing at least one color, the name of the data object with your target area (in the fifth `geom_sf`), the bounding box, and the title and subtitle. 
+```{r}
+ggplot() +
+    geom_sf(data = world, fill = "lightgray") +
+    geom_sf(data = states, fill = "antiquewhite1", size = 0.25) +
+    geom_sf(data = lakes50, fill = "darkblue", color = "black") +
+    geom_sf(data = rivers50, color = "lightblue", size = 0.15) +
+    geom_sf(data = washington, fill = "antiquewhite2", color = "red")+
```

Comment:
You will need to change the "data = washington" to "data = Damitta" to highlight that area. You may also want to change the "fill" to "red" to make it highly visible, since it is relatively small compared to some of the other governorates.